apicast/src/proxy.lua

```diff
@@ -190,8 +190,8 @@ function _M:authorize(service, usage, credentials)
 
     if res.status == 200 then
       if api_keys then
-        ngx.log(ngx.INFO, 'apicast cache write key: ', cached_key)
-        api_keys:set(cached_key, 200)
+        ngx.log(ngx.INFO, 'apicast cache write key: ', cached_key, ', ttl: ', ttl or 'none')
```

Comment:
I think the `none` is not really useful as it prints `nil` otherwise which is already pretty explicit.